hyperledger_matriculation_api.md

```diff
@@ -42,14 +44,23 @@ In the following, the chaincode api is described. It is identical to the hyperle
     
        - Description: This error is returned, if the given parameters could not be parsed. If the string could not be parsed as json, the list of invalidParams is empty. If some attributes within the json are not well formatted, they are listed in invalidParams.
 
+    - ```json
+      {
+        "type": "hl: unprocessable entity",
+        "title": "The given parameters do not conform to the specified format."
```

Comment:
Should we find a different type and title for this one, so that the type dictates whether there invalidParams attribute exists or not?

Reply:
In general it would be nice to have things separated, e.g. if we get an error which provides a list of invalis parameters, there should be invalid parameters.

Reply:
if I recall correctly we agreed on using a 1:1 specification for types.
A single typ always has the exact same title, as well as the same state of "is there an element 'invalid params' " or is there no such element